IBAnimatable/ExplodeAnimator.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  ExplodeAnimator.swift
+//  IBAnimatableApp
+//
+//  Created by Tom Baranes on 03/04/16.
+//  Copyright © 2016 Jake Lin. All rights reserved.
+//
+
+import UIKit
+
+public class ExplodeAnimator: NSObject, AnimatedTransitioning {
+  // MARK: - AnimatorProtocol
+  public var transitionAnimationType: TransitionAnimationType
+  public var transitionDuration: Duration = defaultTransitionDuration
+  public var reverseAnimationType: TransitionAnimationType?
+  public var interactiveGestureType: InteractiveGestureType?
+  
+  init(transitionDuration: Duration) {
+    self.transitionDuration = transitionDuration
+    self.transitionAnimationType = .Explode
+    self.reverseAnimationType = .Explode
+    super.init()
+  }
+}
+
+extension ExplodeAnimator: UIViewControllerAnimatedTransitioning {
+  public func transitionDuration(transitionContext: UIViewControllerContextTransitioning?) -> NSTimeInterval {
+    return retrieveTransitionDuration(transitionContext)
+  }
+  
+  public func animateTransition(transitionContext: UIViewControllerContextTransitioning) {
+    let (tempfromView, tempToView, tempContainerView) = retrieveViews(transitionContext)
+    guard let fromView = tempfromView, toView = tempToView, containerView = tempContainerView else {
+      transitionContext.completeTransition(true)
+      return
+    }
+    
+    containerView.addSubview(toView)
+    containerView.sendSubviewToBack(toView)
```

Comment:
I think we can do `containerView.insertSubview(toView, atIndex: 0)`, but may not have any difference.